docs/user/alerting/troubleshooting/alerting-common-issues.asciidoc

```diff
@@ -266,3 +266,23 @@ This error happens when the `xpack.encryptedSavedObjects.encryptionKey` value us
 | Generate a new API key for the rule by disabling then enabling the rule.
 
 |===
+
+[float]
+[[known-issue-upgrade-rule]]
+==== Rules stop running after upgrade
+
+*Problem*:
+
+Alerting rules that were created or edited in 8.2 stop running after you upgrade
+to later releases. The following error occurs:
+
+[source,text]
+----
+<rule-type>:<UUID>: execution failed - security_exception: [security_exception] Reason: missing authentication credentials for REST request [/_security/user/_has_privileges], caused by: ""
+----
+
+*Solution*:
+
+Upgrade to 8.3.2 or later releases, then go to *{stack-manage-app} > {rules-ui}* and multi-select the failed rules. Choose
```

Comment:
We didn't add the ability to bulk update API keys until [8.5](https://github.com/elastic/kibana/pull/139036) so users in 8.3.2 - 8.5 will have to bulk update individually.

Reply:
Thanks for the clarification!  I'll update that statement in the backports.